src/api/model/modelApi.ts

```diff
@@ -14,57 +14,91 @@
  * under the License.
  */
 
-import { TransactionApi } from '../transaction/transactionApi';
-import {
-  InstallAction,
-  ListSourceAction,
-  Model,
-  ModifyWorkspaceAction,
-} from '../transaction/types';
+import _ from 'lodash';
 
-export class ModelApi extends TransactionApi {
+import { ExecAsyncApi } from '../query/execAsyncApi';
+import { Model } from '../transaction/types';
+export class ModelApi extends ExecAsyncApi {
   async installModels(database: string, engine: string, models: Model[]) {
-    const action: InstallAction = {
-      type: 'InstallAction',
-      sources: models,
-    };
+    const queries = models.map(model => {
+      return `def insert:rel:catalog:model["${model.name}"] = """ ${model.value} """`;
```

Comment:
Hm, shouldn't we do it via query inputs?

Reply:
query inputs are an option but I think those are v1 related

Reply:
Not exactly. They're using v1 data structure at the moment. But they're part of the v2 api as well. https://github.com/RelationalAI/rai-sdk-javascript/blob/main/src/api/query/execAsyncApi.ts#L61